examples/wines.get.json

```diff
@@ -282,6 +282,28 @@
                         "score": 1
                     }
                 ],
+                "flavors": [
+                    {
+                        "id": "acid",
+                        "score": 1
```

Comment:
For food matches wie include the titles. But there are benefits to not include it (we could set up the model `WineCharacteristic` without reference to `WineCharacteristicScores`. Should we go with this approach, I recommend renaming `id` to `characteristic` (and maybe `score` to `value`).

Reply:
The on top `flavor` will be replaced with `characteristic` .. therefore `id` has to remain.

Reply:
> therefore `id` has to remain.

It has to do how we set up the model, and not wether it's called `flavor` or `characteristic`.
The data here represents `WineCharacteristicScore` for which I suggest `characteristic` and `value` as properties.

Reply:
See this draft for details: https://bitbucket.org/detailnet/denner-common/pull-requests/228/add-models-for-wine-characteristics-denner

Reply:
👍  done